src/map/utils/charutils.cpp

```diff
@@ -347,12 +347,12 @@ namespace charutils
      *                                                                       *
      ************************************************************************/
 
-    CCharEntity* LoadChar(uint32 charId)
+    std::unique_ptr<CCharEntity> LoadChar(uint32 charId)
```

Comment:
This would become: `auto LoadChar(uint32 charId) -> std::unique_ptr<CCharEntity>`